CRM/Activity/BAO/Activity.php

```diff
@@ -840,10 +808,10 @@ public static function getActivities($params, $getCount = FALSE) {
         else {
           $activities[$id][$expectedName] = CRM_Utils_Array::value($apiKey, $activity);
           if ($apiKey == 'activity_type_id') {
-            $activities[$id]['activity_type'] = CRM_Utils_Array::value($activities[$id][$expectedName], $activityTypes);
+            $activities[$id]['activity_type'] = CRM_Core_PseudoConstant::getName(__CLASS__, 'activity_type_id', $activities[$id][$expectedName]);
```

Comment:
I think it's better to put in the class name in case the function ever gets moved

Reply:
Ok, changed all instances